README.md

```diff
@@ -964,6 +964,69 @@ Returns a new arrow with the given *data* and *options*.
+
+For example, two quantitative dimensions make a scatterplot:
+
+```js
+Plot.auto(olympians, {x: "height", y: "weight"}).plot()
+// equivalent to Plot.dot(olympians, {x: "height", y: "weight"}).plot()
+```
+
+A monotonic quantitative dimension and another numeric one make a line chart:
+
+```js
+Plot.auto(aapl, {x: "Date", y: "Close"}).plot()
+// equivalent to Plot.lineY(aapl, {x: "Date", y: "Close"}).plot()
+```
+
+One quantitative dimension makes a histogram:
+
+```js
+Plot.auto(penguins, {x: "body_mass_g"}).plot()
+// equivalent to Plot.rectY(penguins, Plot.binX({y: "count"}, {x: "body_mass_g"})).plot()
+```
+
+One ordinal dimension makes a bar chart:
+
+```js
+Plot.auto(penguins, {x: "island"}).plot()
+// equivalent to Plot.barY(penguins, Plot.groupX({y: "count"}, {x: "island"})).plot()
+```
+
+Opinionated inferences also make it easier to switch which dimensions you’re showing by changing only one thing in the code: you can switch a vertical bar chart to a horizontal one by just switching x to y, instead of also having to switch Plot.barY to Plot.barX and Plot.groupX to Plot.groupY.
+
+The options are six channels and a mark override. You must specify either x or y; all others are optional:
+* **x** - corresponds to each mark’s _x_ channel; if specified without _y_, bins or groups on _x_ and shows the count on _y_
+* **y** - corresponds to each mark’s _y_ channel; if specified without _x_, bins or groups on _y_ and shows the count on _x_
+* **fx** - corresponds to each mark’s _fx_ channel
+* **fy** - corresponds to each mark’s _fy_ channel
+* **color** - corresponds to stroke (for line, rule, dot) or fill (for area, rect, bar, cell)
+* **size** - corresponds to the dot’s _r_ channel; setting this always results in a dot mark
+* **mark** - dot, line, area, rule, or bar; each option except dot includes x and y variants, and bar tries picks the appropriate mark from barX, barY, rectX, rectY, rect, or cell
+
+The six channels take one of the following:
+* a string; if not a color name or reducer name, interpreted as a field name
+* an accessor function
+* an object _{value, reduce, color}_, where _value_ is a field string or accessor, _reduce_ is a reducer name or function, and color is the name of a color
+
+Setting a reducer on **x** or **y** implicitly groups or bins on the other (y or x). Setting a reducer on **color** or **size** groups or bins in both x and y dimensions. Setting a reducer on both x and y throws an error.
+
+#### Plot.auto(*data*, *options*)
+
+<!-- jsdoc auto -->
+
+```js
+Plot.auto(athletes, {x: "height", y: "weight", color: "count"})
+// equivalent to Plot.rect(athletes, Plot.bin({fill: "count"}, {x: "height", y: "weight"})).plot()
+```
+
+Returns an automatically selected mark with the given *data* and *options*.
```

Comment:
```suggestion
Returns an automatically selected mark with the given *data* and *options* for a quick view of the data. The heuristic for the choice may evolve in the future.
```

Reply:
Committed! I was wondering, is this the part that shows up in the JSDoc-driven hint popups?

Reply:
Yes: everything between the start and end comments (resp. `<!-- jsdoc xxx -->` and  `<!-- jsdocEnd xxx -->`) gets copied where the the source has `/** @jsdoc xxx */`.

You can see it in the files produced by `yarn prepublishOnly`: dist/marks/dot.js has the jsdoc comment copied from the `<!-- jsdoc dot -->` section of the README file.

Concretely this means that we need to add `/** @jsdoc auto */` in src/marks/auto.js, just above the function definition!